### ISSUE TITLE
Roku app crashes when trying to play "Additional Part" of films

### ISSUE THREAD

**Issue by abudden** [jellyfin-archive/jellyfin-roku-legacy#1658]
## Software Versions

- Jellyfin Server Version: 10.8.9
- Roku Client Version:  2.0 build 1

## Describe the bug

Trying to play the "additional part" of a film (movie) causes the app to immediately crash back to the Roku home-screen with no error message.  This happens every time so is easy to reproduce.

## How To Reproduce

1. Add a film to the library with two mp4 files, one with the filename ending in " cd1.mp4" and one with a file ending in " cd2.mp4" (the files can be identical for the purposes of testing).
2. Update the library so that it is detected by Jellyfin
3. Go to the Jellyfin server webpage to check that it works there: you should be able to play the film and also play CD2, which will appear under "Additional Parts".  All good so far.
4. Now try doing the same on the Roku app.  Part 1 plays fine, Part 2 causes the Jellyfin Roku app to exit immediately with no error message.

I've tried this with several different films and it is consistent.  Any time I try to play the additional part of any film, it crashes.

Swapping MP4 files around thus makes no difference, so it can't be a file issue:

```bash
mv "Filmname (Year) cd1.mp4" temp.mp4
mv "Filmname (Year) cd2.mp4" "Filmname (Year) cd1.mp4"
mv temp.mp4 "Filmname (Year) cd2.mp4"
```

## Expected behaviour

Part 2 should play, just like it does on the web interface.  The app definitely shouldn't crash.

## Logs

Log captured while starting the Roku app, navigating to the film, selecting the additional part, pressing the OK button on the remote and watching it crash:

```
[2024-01-15 17:34:19.932 +00:00] [INF] [44] Emby.Server.Implementations.Session.SessionWebSocketListener: Sending ForceKeepAlive message to 1 inactive WebSockets.
[2024-01-15 17:34:31.933 +00:00] [INF] [106] Emby.Server.Implementations.Session.SessionWebSocketListener: Sending ForceKeepAlive message to 1 inactive WebSockets.
[2024-01-15 17:34:33.004 +00:00] [WRN] [47] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from "http://192.168.3.4:8096/Users/15824df9e78e4c29bff1d8592b9c048c" to "192.168.3.11" in 0:00:00.5817233 with Status Code 200
[2024-01-15 17:34:43.321 +00:00] [WRN] [92] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from "http://192.168.3.4:8096/Items/c9b54f0f10d37fa62e52788103d29752/Images/Primary?maxHeight=384&maxWidth=196&quality=90" to "192.168.3.11" in 0:00:00.748827 with Status Code 200
```

## Connection Information

- Local (ethernet cables to switch, so no wifi involved)
- http

## Screenshots

![image](https://github.com/jellyfin/jellyfin-roku/assets/670607/aa252870-6fba-47a4-878d-bf3912ee6c84)

![image](https://github.com/jellyfin/jellyfin-roku/assets/670607/cc6de5ea-be16-47be-90e9-bb4644147a85)

![image](https://github.com/jellyfin/jellyfin-roku/assets/670607/62ef2987-298f-4f02-9be3-e71cef6c281a)

**Comment by macmini888** [comment]
I tested this on my Roku Ultra 4660X and am having the same issue.

**Comment by 1hitsong** [comment]
Recreated on my dev setup.

**Comment by 1hitsong** [comment]
Fixed by #1678
